### ISSUE TITLE
flutter_inappwebview/flutter_inappwebview-Swift.h' file not found

### ISSUE THREAD

**Issue by rchtgpt** [flutter/flutter#77352]
## Steps to Reproduce

1. install ` flutter_inappwebview: 4.0.0+4`.
2. run the application on an iOS simulator to reproduce the error.

**Expected results:** 
App should compile normally, as it does on android.

**Actual results:**
```
fatal error: 'flutter_inappwebview/flutter_inappwebview-Swift.h' file not found
    #import <flutter_inappwebview/flutter_inappwebview-Swift.h>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Could not build the application for the simulator.
Error launching application on iPhone 12 Pro.
```

<details>
  <summary>Logs</summary>

```
> flutter run --verbose
https://pastebin.com/GTUh552e
```

```
❯ flutter analyze
Analyzing kyt-mobile...                                                 

   info • Unused import: 'dart:developer' • lib/functions/getTextFromOCR.dart:2:8 • unused_import
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/functions/getTextFromOCR.dart:19:28 • deprecated_member_use
   info • 'List' is deprecated and shouldn't be used • lib/screens/archivedRecords.dart:52:56 • deprecated_member_use
   info • The value of the local variable 'snapshot' isn't used • lib/screens/editProfile.dart:227:53 • unused_local_variable
   info • The value of the local variable 'snapshot' isn't used • lib/screens/register.dart:376:55 • unused_local_variable
   info • The value of the field '_webViewController' isn't used • lib/screens/reportABug.dart:14:26 • unused_field
   info • This class (or a class that this class inherits from) is marked as '@immutable', but one or more of its instance fields aren't
          final: Upload.testNameContext • lib/screens/upload.dart:20:7 • must_be_immutable
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:112:51 • deprecated_member_use
   info • 'pickImage' is deprecated and shouldn't be used. Use imagePicker.getImage() method instead. • lib/screens/upload.dart:126:65 •
          deprecated_member_use
   info • The value of the local variable 'snapshot' isn't used • lib/screens/upload.dart:149:55 • unused_local_variable
   info • The value of the local variable 'ocrText' isn't used • lib/screens/upload.dart:173:30 • unused_local_variable
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:200:50 • deprecated_member_use
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:215:50 • deprecated_member_use
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:230:51 • deprecated_member_use
   info • 'pickImage' is deprecated and shouldn't be used. Use imagePicker.getImage() method instead. • lib/screens/upload.dart:237:43 •
          deprecated_member_use
   info • The value of the local variable 'snapshot' isn't used • lib/screens/upload.dart:259:53 • unused_local_variable
   info • The value of the local variable 'ocrText' isn't used • lib/screens/upload.dart:283:28 • unused_local_variable
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:310:48 • deprecated_member_use
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:324:48 • deprecated_member_use
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:345:49 • deprecated_member_use
   info • 'pickImage' is deprecated and shouldn't be used. Use imagePicker.getImage() method instead. • lib/screens/upload.dart:359:64 •
          deprecated_member_use
   info • The value of the local variable 'snapshot' isn't used • lib/screens/upload.dart:382:53 • unused_local_variable
   info • The value of the local variable 'ocrText' isn't used • lib/screens/upload.dart:406:28 • unused_local_variable
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:433:48 • deprecated_member_use
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:447:48 • deprecated_member_use
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:461:49 • deprecated_member_use
   info • 'pickImage' is deprecated and shouldn't be used. Use imagePicker.getImage() method instead. • lib/screens/upload.dart:468:41 •
          deprecated_member_use
   info • The value of the local variable 'snapshot' isn't used • lib/screens/upload.dart:490:51 • unused_local_variable
   info • The value of the local variable 'ocrText' isn't used • lib/screens/upload.dart:514:26 • unused_local_variable
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:541:46 • deprecated_member_use
   info • 'showSnackBar' is deprecated and shouldn't be used. Use ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/screens/upload.dart:555:46 • deprecated_member_use

31 issues found. (ran in 2.5s)
```

```
❯ flutter doctor -v
[✓] Flutter (Channel beta, 2.0.1, on macOS 11.1 20C69 darwin-x64, locale en-IN)
    • Flutter version 2.0.1 at /Users/rachitgupta/dev/flutter
    • Framework revision c5a4b4029c (18 hours ago), 2021-03-04 09:47:48 -0800
    • Engine revision 40441def69
    • Dart version 2.12.0

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/rachitgupta/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 11.1 20C69 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 88.0.4324.192

! Doctor found issues in 1 category.
```

</details>

**Comment by rchtgpt** [comment]
tried everything on #32910 but couldn't make it work.

**Comment by TahaTesser** [comment]
Hi @rchtgpt 
First of all, please upgrade to the latest `stable` or `beta` channel
`flutter channel beta`
`flutter upgrade --force`
`flutter doctor -v`
and try this [solution](https://github.com/flutter/flutter/issues/70895#issuecomment-744734693) 
Thank you

**Comment by rchtgpt** [comment]
> Hi @rchtgpt
> First of all, please upgrade to the latest `stable` or `beta` channel
> `flutter channel beta`
> `flutter upgrade --force`
> `flutter doctor -v`
> and try this [solution](https://github.com/flutter/flutter/issues/70895#issuecomment-744734693)
> Thank you

Hi @TahaTesser, I have upgraded to the latest beta channel and have updated the `flutter doctor -v` output in the issue comment. About the [solution](https://github.com/flutter/flutter/issues/70895#issuecomment-744734693), it doesn't work and gives the same error message.

<details>
<summary>log</summary>

```
Running "flutter pub get" in kyt-mobile...
Launching lib/main.dart on iPhone 12 Pro in debug mode...
Running pod install...
Running Xcode build...
Xcode build done.                                           14.4s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/url_launcher-5.7.10/ios/Classes/FLTURLLauncherPlugin.m:153:57: warning: 'keyWindow' is deprecated: first deprecated in iOS 13.0 - Should not be used for applications that support multiple scenes as it returns a key window across all connected scenes [-Wdeprecated-declarations]
                                                           .keyWindow.rootViewController];
                                                            ^
    In module 'UIKit' imported from /Users/rachitgupta/Documents/trial/2020/kyt-mobile/ios/Pods/Target Support Files/url_launcher/url_launcher-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:134:51: note: 'keyWindow' has been explicitly marked deprecated here
    @property(nullable, nonatomic,readonly) UIWindow *keyWindow API_DEPRECATED("Should not be used for applications that support multiple scenes as it returns a key window across all connected scenes", ios(2.0, 13.0));
                                                      ^
    1 warning generated.
    2 warnings generated.
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/image_picker-0.6.7+22/ios/Classes/FLTImagePickerPhotoAssetUtil.m:17:44: warning: 'UIImagePickerControllerReferenceURL' is deprecated: first deprecated in iOS 11.0 - Will be removed in a future release, use PHPicker. [-Wdeprecated-declarations]
      NSURL *referenceURL = [info objectForKey:UIImagePickerControllerReferenceURL];
                                               ^
    In module 'UIKit' imported from /Users/rachitgupta/Documents/trial/2020/kyt-mobile/ios/Pods/Target Support Files/image_picker/image_picker-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIImagePickerController.h:62:51: note: 'UIImagePickerControllerReferenceURL' has been explicitly marked deprecated here
    UIKIT_EXTERN UIImagePickerControllerInfoKey const UIImagePickerControllerReferenceURL API_DEPRECATED("Will be removed in a future release, use PHPicker.", ios(4.1, 11.0)) API_UNAVAILABLE(tvos); // an NSURL that references an asset in the AssetsLibrary framework
                                                      ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/image_picker-0.6.7+22/ios/Classes/FLTImagePickerPhotoAssetUtil.m:21:47: warning: 'fetchAssetsWithALAssetURLs:options:' is deprecated: first deprecated in iOS 11 - Will be removed in a future release [-Wdeprecated-declarations]
      PHFetchResult<PHAsset *> *result = [PHAsset fetchAssetsWithALAssetURLs:@[ referenceURL ]
                                                  ^
    In module 'Photos' imported from /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/image_picker-0.6.7+22/ios/Classes/FLTImagePickerPhotoAssetUtil.h:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.4.sdk/System/Library/Frameworks/Photos.framework/Headers/PHAsset.h:74:1: note: 'fetchAssetsWithALAssetURLs:options:' has been explicitly marked deprecated here
    + (PHFetchResult<PHAsset *> *)fetchAssetsWithALAssetURLs:(NSArray<NSURL *> *)assetURLs options:(nullable PHFetchOptions *)options API_DEPRECATED("Will be removed in a future release", ios(8, 11), tvos(8, 11)) API_UNAVAILABLE(macos);
    ^
    2 warnings generated.
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+17/ios/Classes/CameraPlugin.m:61:1: warning: implementing deprecated method [-Wdeprecated-implementations]
    - (void)captureOutput:(AVCapturePhotoOutput *)output
    ^
    In module 'AVFoundation' imported from /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+17/ios/Classes/CameraPlugin.m:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.4.sdk/System/Library/Frameworks/AVFoundation.framework/Headers/AVCapturePhotoOutput.h:753:1: note: method 'captureOutput:didFinishProcessingPhotoSampleBuffer:previewPhotoSampleBuffer:resolvedSettings:bracketSettings:error:' declared here
    - (void)captureOutput:(AVCapturePhotoOutput *)output didFinishProcessingPhotoSampleBuffer:(nullable CMSampleBufferRef)photoSampleBuffer previewPhotoSampleBuffer:(nullable CMSampleBufferRef)previewPhotoSampleBuffer resolvedSettings:(AVCaptureResolvedPhotoSettings *)resolvedSettings bracketSettings:(nullable AVCaptureBracketedStillImageSettings *)bracketSettings error:(nullable NSError *)error API_DEPRECATED_WITH_REPLACEMENT("-captureOutput:didFinishProcessingPhoto:error:", ios(10.0, 11.0)) API_UNAVAILABLE(macos);
    ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+17/ios/Classes/CameraPlugin.m:73:7: warning: 'JPEGPhotoDataRepresentationForJPEGSampleBuffer:previewPhotoSampleBuffer:' is deprecated: first deprecated in iOS 11.0 [-Wdeprecated-declarations]
          JPEGPhotoDataRepresentationForJPEGSampleBuffer:photoSampleBuffer
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          -[AVCapturePhoto fileDataRepresentation]
    In module 'AVFoundation' imported from /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+17/ios/Classes/CameraPlugin.m:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.4.sdk/System/Library/Frameworks/AVFoundation.framework/Headers/AVCapturePhotoOutput.h:544:1: note: 'JPEGPhotoDataRepresentationForJPEGSampleBuffer:previewPhotoSampleBuffer:' has been explicitly marked deprecated here
    + (nullable NSData *)JPEGPhotoDataRepresentationForJPEGSampleBuffer:(CMSampleBufferRef)JPEGSampleBuffer previewPhotoSampleBuffer:(nullable CMSampleBufferRef)previewPhotoSampleBuffer API_DEPRECATED_WITH_REPLACEMENT("-[AVCapturePhoto fileDataRepresentation]", ios(10.0, 11.0)) API_UNAVAILABLE(macos);
    ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+17/ios/Classes/CameraPlugin.m:339:13: warning: 'OSAtomicCompareAndSwapPtrBarrier' is deprecated: first deprecated in iOS 10.0 - Use atomic_compare_exchange_strong() from <stdatomic.h> instead [-Wdeprecated-declarations]
        while (!OSAtomicCompareAndSwapPtrBarrier(old, newBuffer, (void **)&_latestPixelBuffer)) {
                ^
    In module 'Darwin' imported from /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+17/ios/Classes/CameraPlugin.m:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.4.sdk/usr/include/libkern/OSAtomicDeprecated.h:548:6: note: 'OSAtomicCompareAndSwapPtrBarrier' has been explicitly marked deprecated here
    bool    OSAtomicCompareAndSwapPtrBarrier( void *__oldValue, void *__newValue, void * volatile *__theValue );
            ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+17/ios/Classes/CameraPlugin.m:561:11: warning: 'OSAtomicCompareAndSwapPtrBarrier' is deprecated: first deprecated in iOS 10.0 - Use atomic_compare_exchange_strong() from <stdatomic.h> instead [-Wdeprecated-declarations]
      while (!OSAtomicCompareAndSwapPtrBarrier(pixelBuffer, nil, (void **)&_latestPixelBuffer)) {
              ^
    In module 'Darwin' imported from /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+17/ios/Classes/CameraPlugin.m:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.4.sdk/usr/include/libkern/OSAtomicDeprecated.h:548:6: note: 'OSAtomicCompareAndSwapPtrBarrier' has been explicitly marked deprecated here
    bool    OSAtomicCompareAndSwapPtrBarrier( void *__oldValue, void *__newValue, void * volatile *__theValue );
            ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+17/ios/Classes/CameraPlugin.m:679:36: warning: 'AVVideoCodecH264' is deprecated: first deprecated in iOS 11.0 [-Wdeprecated-declarations]
          dictionaryWithObjectsAndKeys:AVVideoCodecH264, AVVideoCodecKey,
                                       ^~~~~~~~~~~~~~~~
                                       AVVideoCodecTypeH264
    In module 'AVFoundation' imported from /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/camera-0.5.8+17/ios/Classes/CameraPlugin.m:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.4.sdk/System/Library/Frameworks/AVFoundation.framework/Headers/AVVideoSettings.h:57:29: note: 'AVVideoCodecH264' has been explicitly marked deprecated here
            AVF_EXPORT NSString *const AVVideoCodecH264 /* @"avc1" */                               API_DEPRECATED_WITH_REPLACEMENT("AVVideoCodecTypeH264", macos(10.7, 10.13), ios(4.0, 11.0), tvos(9.0, 11.0)) API_UNAVAILABLE(watchos);
                                       ^
    5 warnings generated.
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/FlutterWebViewController.swift:107:34: warning: 'keyWindow' was deprecated in iOS 13.0: Should not be used for applications that support multiple scenes as it returns a key window across all connected scenes
                UIApplication.shared.keyWindow!.insertSubview(self.myView!, at: 0)
                                     ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserManager.swift:91:64: warning: 'statusBarStyle' was deprecated in iOS 13.0: Use the statusBarManager property of the window scene instead.
                self.previousStatusBarStyle = UIApplication.shared.statusBarStyle.rawValue
                                                                   ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserManager.swift:98:52: warning: 'keyWindow' was deprecated in iOS 13.0: Should not be used for applications that support multiple scenes as it returns a key window across all connected scenes
            let baseWindowLevel = UIApplication.shared.keyWindow?.windowLevel
                                                       ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: warning: coercion of implicitly unwrappable value of type 'InAppWebView_IBWrapper?' to 'Any' does not unwrap optional
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:26:19: note: implicitly unwrapped property 'containerWebView' declared here
        @IBOutlet var containerWebView: InAppWebView_IBWrapper!
                      ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: note: provide a default value to avoid this warning
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
                                       ?? <#default value#>
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: note: force-unwrap the value to avoid this warning
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
                                      !
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: note: explicitly cast to 'Any' with 'as Any' to silence this warning
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
                                       as Any
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:586:56: warning: 'keyWindow' was deprecated in iOS 13.0: Should not be used for applications that support multiple scenes as it returns a key window across all connected scenes
                let baseWindowLevel = UIApplication.shared.keyWindow?.windowLevel
                                                           ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:603:42: warning: 'statusBarStyle' was deprecated in iOS 13.0: Use the statusBarManager property of the window scene instead.
                        UIApplication.shared.statusBarStyle = UIStatusBarStyle(rawValue: self.previousStatusBarStyle)!
                                             ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:675:59: warning: 'statusBarFrame' was deprecated in iOS 13.0: Use the statusBarManager property of the window scene instead.
            let statusBarFrame: CGRect = UIApplication.shared.statusBarFrame
                                                              ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:705:17: warning: 'scanHexInt32' was deprecated in iOS 13.0
            scanner.scanHexInt32(&hexInt)
                    ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:788:34: warning: 'statusBarStyle' was deprecated in iOS 13.0: Use the statusBarManager property of the window scene instead.
                UIApplication.shared.statusBarStyle = UIStatusBarStyle(rawValue: previousStatusBarStyle)!
                                     ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:1278:25: warning: 'spotlightSuggestion' was deprecated in iOS 10.0: renamed to 'WKDataDetectorTypes.lookupSuggestion'
                    return .spotlightSuggestion
                            ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:1278:25: note: use 'WKDataDetectorTypes.lookupSuggestion' instead
                    return .spotlightSuggestion
                            ^~~~~~~~~~~~~~~~~~~
                            WKDataDetectorTypes.lookupSuggestion
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:1313:31: warning: 'spotlightSuggestion' was deprecated in iOS 10.0: renamed to 'WKDataDetectorTypes.lookupSuggestion'
                if type.contains(.spotlightSuggestion) {
                                  ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:1313:31: note: use 'WKDataDetectorTypes.lookupSuggestion' instead
                if type.contains(.spotlightSuggestion) {
                                  ^~~~~~~~~~~~~~~~~~~
                                  WKDataDetectorTypes.lookupSuggestion
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:2707:9: warning: 'SecTrustEvaluate' was deprecated in iOS 13.0: renamed to 'SecTrustEvaluateWithError(_:_:)'
            SecTrustEvaluate(serverTrust, &secResult);
            ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:2707:9: note: use 'SecTrustEvaluateWithError(_:_:)' instead
            SecTrustEvaluate(serverTrust, &secResult);
            ^~~~~~~~~~~~~~~~
            SecTrustEvaluateWithError
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: warning: coercion of implicitly unwrappable value of type 'InAppWebView_IBWrapper?' to 'Any' does not unwrap optional
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:26:19: note: implicitly unwrapped property 'containerWebView' declared here
        @IBOutlet var containerWebView: InAppWebView_IBWrapper!
                      ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: note: provide a default value to avoid this warning
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
                                       ?? <#default value#>
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: note: force-unwrap the value to avoid this warning
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
                                      !
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:388:19: note: explicitly cast to 'Any' with 'as Any' to silence this warning
                print(containerWebView)
                      ^~~~~~~~~~~~~~~~
                                       as Any
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:586:56: warning: 'keyWindow' was deprecated in iOS 13.0: Should not be used for applications that support multiple scenes as it returns a key window across all connected scenes
                let baseWindowLevel = UIApplication.shared.keyWindow?.windowLevel
                                                           ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:603:42: warning: 'statusBarStyle' was deprecated in iOS 13.0: Use the statusBarManager property of the window scene instead.
                        UIApplication.shared.statusBarStyle = UIStatusBarStyle(rawValue: self.previousStatusBarStyle)!
                                             ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:675:59: warning: 'statusBarFrame' was deprecated in iOS 13.0: Use the statusBarManager property of the window scene instead.
            let statusBarFrame: CGRect = UIApplication.shared.statusBarFrame
                                                              ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:705:17: warning: 'scanHexInt32' was deprecated in iOS 13.0
            scanner.scanHexInt32(&hexInt)
                    ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppBrowserWebViewController.swift:788:34: warning: 'statusBarStyle' was deprecated in iOS 13.0: Use the statusBarManager property of the window scene instead.
                UIApplication.shared.statusBarStyle = UIStatusBarStyle(rawValue: previousStatusBarStyle)!
                                     ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:1278:25: warning: 'spotlightSuggestion' was deprecated in iOS 10.0: renamed to 'WKDataDetectorTypes.lookupSuggestion'
                    return .spotlightSuggestion
                            ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:1278:25: note: use 'WKDataDetectorTypes.lookupSuggestion' instead
                    return .spotlightSuggestion
                            ^~~~~~~~~~~~~~~~~~~
                            WKDataDetectorTypes.lookupSuggestion
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:1313:31: warning: 'spotlightSuggestion' was deprecated in iOS 10.0: renamed to 'WKDataDetectorTypes.lookupSuggestion'
                if type.contains(.spotlightSuggestion) {
                                  ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:1313:31: note: use 'WKDataDetectorTypes.lookupSuggestion' instead
                if type.contains(.spotlightSuggestion) {
                                  ^~~~~~~~~~~~~~~~~~~
                                  WKDataDetectorTypes.lookupSuggestion
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:2707:9: warning: 'SecTrustEvaluate' was deprecated in iOS 13.0: renamed to 'SecTrustEvaluateWithError(_:_:)'
            SecTrustEvaluate(serverTrust, &secResult);
            ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebView.swift:2707:9: note: use 'SecTrustEvaluateWithError(_:_:)' instead
            SecTrustEvaluate(serverTrust, &secResult);
            ^~~~~~~~~~~~~~~~
            SecTrustEvaluateWithError
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/MyCookieManager.swift:120:16: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
                if #available(iOS 13.0, *) {
                   ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/MyCookieManager.swift:12:7: note: enclosing scope here
    class MyCookieManager: NSObject, FlutterPlugin {
          ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/MyCookieManager.swift:149:24: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
                        if #available(iOS 13.0, *) {
                           ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/MyCookieManager.swift:12:7: note: enclosing scope here
    class MyCookieManager: NSObject, FlutterPlugin {
          ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariBrowserOptions.swift:29:16: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
                if #available(iOS 11.0, *) {
                   ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariBrowserOptions.swift:12:14: note: enclosing scope here
    public class SafariBrowserOptions: Options<SafariViewController> {
                 ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:58:12: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
            if #available(iOS 11.0, *) {
               ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:12:14: note: enclosing scope here
    public class SafariViewController: SFSafariViewController, FlutterPlugin, SFSafariViewControllerDelegate {
                 ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:62:12: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
            if #available(iOS 10.0, *) {
               ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:12:14: note: enclosing scope here
    public class SafariViewController: SFSafariViewController, FlutterPlugin, SFSafariViewControllerDelegate {
                 ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:58:12: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
            if #available(iOS 11.0, *) {
               ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:12:14: note: enclosing scope here
    public class SafariViewController: SFSafariViewController, FlutterPlugin, SFSafariViewControllerDelegate {
                 ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:62:12: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
            if #available(iOS 10.0, *) {
               ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:12:14: note: enclosing scope here
    public class SafariViewController: SFSafariViewController, FlutterPlugin, SFSafariViewControllerDelegate {
                 ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:162:17: warning: 'scanHexInt32' was deprecated in iOS 13.0
            scanner.scanHexInt32(&hexInt)
                    ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariBrowserOptions.swift:29:16: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
                if #available(iOS 11.0, *) {
                   ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariBrowserOptions.swift:12:14: note: enclosing scope here
    public class SafariBrowserOptions: Options<SafariViewController> {
                 ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:58:12: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
            if #available(iOS 11.0, *) {
               ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:12:14: note: enclosing scope here
    public class SafariViewController: SFSafariViewController, FlutterPlugin, SFSafariViewControllerDelegate {
                 ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:62:12: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
            if #available(iOS 10.0, *) {
               ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:12:14: note: enclosing scope here
    public class SafariViewController: SFSafariViewController, FlutterPlugin, SFSafariViewControllerDelegate {
                 ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:58:12: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
            if #available(iOS 11.0, *) {
               ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:12:14: note: enclosing scope here
    public class SafariViewController: SFSafariViewController, FlutterPlugin, SFSafariViewControllerDelegate {
                 ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:62:12: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
            if #available(iOS 10.0, *) {
               ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:12:14: note: enclosing scope here
    public class SafariViewController: SFSafariViewController, FlutterPlugin, SFSafariViewControllerDelegate {
                 ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/SafariViewController.swift:162:17: warning: 'scanHexInt32' was deprecated in iOS 13.0
            scanner.scanHexInt32(&hexInt)
                    ^
    /Users/rachitgupta/Dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_inappwebview-4.0.0+4/ios/Classes/InAppWebViewFlutterPlugin.m:19:9: fatal error: 'flutter_inappwebview/flutter_inappwebview-Swift.h' file not found
    #import <flutter_inappwebview/flutter_inappwebview-Swift.h>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Could not build the application for the simulator.
Error launching application on iPhone 12 Pro.
```
</details>

**Comment by TahaTesser** [comment]
Hi @rchtgpt 
This is a duplicate of https://github.com/flutter/flutter/issues/76520
There is a possible [solution](https://github.com/flutter/flutter/issues/76520#issuecomment-783337136) 
Please follow up on that issue, I'm closing the current one as a duplicate.
If you disagree, please write in the comments and I will reopen it.
Thank you
